onnxruntime/core/providers/coreml/builders/impl/builder_utils.cc

```diff
@@ -313,6 +313,14 @@ void AddOperationInput(MILSpec::Operation& op, std::string_view input_name, std:
   (*op.mutable_inputs())[input_name] = std::move(arg);
 }
 
+void AddOperationInputs(MILSpec::Operation& op, std::string_view input_name,
+                        const std::vector<std::string_view>& value_names) {
+  MILSpec::Argument& arg = (*op.mutable_inputs())[input_name];
```

Comment:
this is slightly different from `AddOperationInput()` in that it may update an existing input that has the same name instead of replacing it. should they be consistent? or named more distinctly, if not?

Reply:
That's a good point. I assume that inputs will have distinct names. 
@skottmckay 


On the other note, is something like this better?

```
MILSpec::Argument arg;
for (const auto& value : value_names) {
    arg.mutable_arguments()->Add()->set_name(std::string(value));
  }
(*op.mutable_inputs())[input_name] = std::move(arg);

```

Reply:
consistent is good. I don't think there's any need to edit an existing argument. 

fwiw it was done this way when trying to figure out an access violation and whether that was due to having multiple arguments. it did start off consistent because cut-and-pasting is easy. :P